rlog.go

```diff
@@ -603,8 +603,8 @@ func basicLog(logLevel int, traceLevel int, isLocked bool, format string, prefix
 
 	callerInfo := ""
 	if settingShowCallerInfo {
-		callerInfo = fmt.Sprintf("[%s:%d (%s)] ", moduleAndFileName,
-			line, callingFuncName)
+		callerInfo = fmt.Sprintf("[%s:%d (%d:%s)] ", moduleAndFileName,
```

Comment:
This is all fine, except that I wouldn't print the PID right there. The format currently is:

    [foo.go:123 (myFunc)]

Putting the PID in the parenthesis with the function name doesn't seem right. How about this order:

    [12345 foo.go:125 (myFunc)]

? With the PID printed at the start.